tests/test_beta50_helper_production_target_scope.py

```
@@ -449,6 +796,472 @@ async def test_consequential_and_arbitrary_controls_remain_proportional(self):
+        )
+        self.assertTrue(rebuilt)
+        self.target = self.fixture["target_entity_id"]
+
+    def test_capture_provenance_and_sanitized_payload_are_bound(self):
+        provenance = self.fixture["provenance"]
+        canonical = lambda value: json.dumps(
+            value,
+            sort_keys=True,
+            separators=(",", ":"),
+            ensure_ascii=False,
+        ).encode("utf-8")
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `canonical` as a `def`

(E731)